lib/init/rc.boot

```diff
@@ -167,6 +171,16 @@ main() {
     command -v udevd >/dev/null &&
         udevadm control --exit
 
+    log "Running rc.d scripts..."; {
+        if [ -d /etc/rc.d ] ; then
+            set +f
+            for file in /etc/rc.d/*.boot ; do
+                [ -x "$file" ] && . "$file"
```

Comment:
The file doesn't have to executable to be sourced using `.`. I'd change this to `[ -f ]` as we **do** need to check that it exists as the glob can fail.